arithmetization/src/main/java/net/consensys/linea/zktracer/module/hub/section/AccountSection.java

```diff
@@ -46,6 +47,7 @@ public AccountSection(Hub hub) {
     if (hub.opCode().isAnyOf(OpCode.SELFBALANCE, OpCode.CODESIZE)) {
       if (Exceptions.any(exceptions)) {
         // the "squash parent return data" context row is all there is
+        Preconditions.checkArgument(Exceptions.outOfGasException(exceptions));
```

Comment:
This is true because we never enter here in case of a `STACK_OVERFLOW_EXCEPTION`, but I would include it as an explanatory comment.